github/repos_traffic.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// Referrer represent a referrer information.
+type Referrer struct {
+	Referrer *string `json:"referrer,omitempty"`
+	Count    *int    `json:"count,omitempty"`
+	Uniques  *int    `json:"uniques,omitempty"`
+}
+
+// Path represent a referrer information.
+type Path struct {
+	Path    *string `json:"path,omitempty"`
+	Title   *string `json:"title,omitempty"`
+	Count   *int    `json:"count,omitempty"`
+	Uniques *int    `json:"uniques,omitempty"`
+}
+
+// Time represents a time stamp as in a datapoint.
+//
+// This is necessary because github uses milliseconds timestamp.
+type Time struct {
```

Comment:
rather than creating a new type, perhaps we could modify https://godoc.org/github.com/google/go-github/github#Timestamp to handle millisecond precision times.  Right now it handles seconds, but using some pretty basic heuristics, we should be able to determine if a given value is intended to be seconds or milliseconds.  Something like:

```go
if i > 1e9 {
    i /= 1000
}
```

That'll at least work for dates up to the year 2286.  What do you think?